lib/ui/painting.dart

```diff
@@ -4081,6 +4081,26 @@ class FragmentProgram extends NativeFieldWrapperClass1 {
     ));
   }
 
+  static Map<String, WeakReference<FragmentProgram>> _shaderRegistry =
+      <String, WeakReference<FragmentProgram>>{};
+
+  static FragmentProgram fromAsset({
```

Comment:
nit, consider making this a position parameter to match ImmutableBuffer (https://github.com/flutter/engine/blob/main/lib/ui/painting.dart#L5904), and since it is easier to deduce from context since there is only 1 arg

Reply:
Done.